packages/infra/lib/api-stack.ts

```diff
@@ -1080,6 +1095,64 @@ export class APIStack extends Stack {
     return documentDownloaderLambda;
   }
 
+  private setupBulkUrlSigningLambda(ownProps: {
+    lambdaLayers: LambdaLayers;
+    vpc: ec2.IVpc;
+    medicalDocumentsBucket: s3.Bucket;
+    fhirServerUrl: string;
+    envType: EnvType;
+    sentryDsn: string | undefined;
+    alarmAction: SnsAction | undefined;
+    searchEndpoint: string;
+    searchIndex: string;
+    searchUserName: string;
+    searchPassword: string;
+    apiService: ecs_patterns.NetworkLoadBalancedFargateService;
+  }): Lambda {
+    const {
+      lambdaLayers,
+      vpc,
+      medicalDocumentsBucket,
+      fhirServerUrl,
+      sentryDsn,
+      alarmAction,
+      envType,
+      searchEndpoint,
+      searchIndex,
+      searchUserName,
+      searchPassword,
+      apiService,
+    } = ownProps;
+
+    const bulkUrlSigningLambda = createLambda({
+      stack: this,
+      name: "BulkUrlSigning",
+      runtime: lambda.Runtime.NODEJS_18_X,
```

Comment:
I'd leave the runtime out (18 is the default on `createLambda`), so its easier to update when needed.